test/federated/test_federated_client.py

```diff
@@ -61,14 +52,17 @@ def test_set_obj_other():
     assert fed_client._objects[dummy_data.id] == dummy_data
 
 
-def test_fit():
```

Comment:
Can we leave this test as it was, please?
The goal of this test was to have a unit test, that does not need to be skipped (does not depend on jit). 
Also, no workers are used and it was a UNIT test for the FederatedClient only.
I added these tests to get the coverage above 95%.